blockchain/elrondBaseProxy.go

```diff
@@ -9,25 +9,25 @@ import (
 	"sync"
 	"time"
 
-	"github.com/ElrondNetwork/elrond-go-core/core/check"
-	logger "github.com/ElrondNetwork/elrond-go-logger"
-	"github.com/ElrondNetwork/elrond-sdk-erdgo/core"
-	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/multiversx/mx-sdk-go/core"
+	"github.com/multiversx/mx-sdk-go/data"
 )
 
-var log = logger.GetOrCreate("elrond-sdk-erdgo/blockchain")
+var log = logger.GetOrCreate("mx-sdk-go/blockchain")
 
 const (
 	minimumCachingInterval = time.Second
 )
 
-type argsElrondBaseProxy struct {
+type argsMultiversXBaseProxy struct {
 	expirationTime    time.Duration
 	httpClientWrapper httpClientWrapper
 	endpointProvider  EndpointProvider
 }
 
-type elrondBaseProxy struct {
+type multiversXBaseProxy struct {
```

Comment:
maybe this structure can be named `baseProxy` without `multiversX` prefix

Reply:
fixed